src/common/Database/Implementation/CharacterDatabase.cpp

```diff
@@ -559,6 +559,7 @@ void CharacterDatabaseConnection::DoPrepareStatements()
     PrepareStatement(CHAR_INS_PVPSTATS_BATTLEGROUND, "INSERT INTO pvpstats_battlegrounds (id, winner_faction, bracket_id, type, date) VALUES (?, ?, ?, ?, NOW())", CONNECTION_ASYNC);
     PrepareStatement(CHAR_INS_PVPSTATS_PLAYER, "INSERT INTO pvpstats_players (battleground_id, character_guid, winner, score_killing_blows, score_deaths, score_honorable_kills, score_bonus_honor, score_damage_done, score_healing_done, attr_1, attr_2, attr_3, attr_4, attr_5) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)", CONNECTION_ASYNC);
     PrepareStatement(CHAR_SEL_PVPSTATS_FACTIONS_OVERALL, "SELECT winner_faction, COUNT(*) AS count FROM pvpstats_battlegrounds WHERE DATEDIFF(NOW(), date) < 7 GROUP BY winner_faction ORDER BY winner_faction ASC", CONNECTION_SYNCH);
+    PrepareStatement(CHAR_SEL_PVPSTATS_BRACKET_MONTH, "SELECT character_guid, COUNT(character_guid) AS count, characters.name as character_name FROM pvpstats_players INNER JOIN pvpstats_battlegrounds ON pvpstats_players.battleground_id = pvpstats_battlegrounds.id INNER JOIN characters ON pvpstats_players.character_guid = characters.guid WHERE characters.deleteDate IS NULL AND pvpstats_players.winner = 1 AND MONTH(date) = MONTH(NOW()) AND YEAR(date) = YEAR(NOW()) AND bracket_id = ? GROUP BY character_guid ORDER BY count(character_guid) DESC LIMIT 0,?", CONNECTION_SYNCH);
```

Comment:
I suggest the following query, to apply the filters of the WHERE before joining the tables, it should save a lot of copmutational operations
```SQL
SELECT character_guid, COUNT(character_guid) AS count, characters.name as character_name
FROM pvpstats_players
INNER JOIN pvpstats_battlegrounds ON pvpstats_players.battleground_id = pvpstats_battlegrounds.id AND bracket_id = ? AND MONTH(date) = MONTH(NOW()) AND YEAR(date) = YEAR(NOW())
INNER JOIN characters ON pvpstats_players.character_guid = characters.guid AND characters.deleteDate IS NULL
WHERE
  pvpstats_players.winner = 1
GROUP BY character_guid
ORDER BY count(character_guid)
DESC LIMIT 0
```